index.js

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-const hook = require('skygear-chat/hook');
+const skygearChatCloud = require('skygear-chat/cloud');
 const skygearCloud = require('skygear/cloud');
 const container = new skygearCloud.CloudCodeContainer();
+
 container.apiKey = skygearCloud.settings.apiKey;
 container.endPoint = skygearCloud.settings.skygearEndpoint + '/';
```

Comment:
@howawong When I review the py version, I saw we need to set the container user id. Do we have to do it here too? Is the push notification working without setting the container user id?

Reply:
Yes. As discussed, it is working.